src/testing/services/behat-dom.ts

```diff
@@ -390,6 +390,28 @@ export class TestingBehatDomUtilsService {
         return this.getClosestMatching(parent, selector, container);
     }
 
+    /**
+     * For a list of elements, get the closest matching ancestor for each of them, if matches.
+     *
+     * @param elements List of elements.
+     * @param selector Ancestor selector.
+     * @param container Topmost container to search within.
+     * @returns Closest matching elements.
+     */
+    protected getClosestMatchingElements(elements: HTMLElement[], selector: string, container: HTMLElement): HTMLElement[] {
+        let closestMatchingElements: HTMLElement[] = elements;
+
+        closestMatchingElements = [];
```

Comment:
You're still assigning the variable twice, it can be simplified to:

```suggestion
        let closestMatchingElements: HTMLElement[] = [];
```